### PR TITLE
docs: Use console type for interactive shell snippets

### PR DESCRIPTION
**Pull request by mkmik**
minor doc rendering improvement

### CODE REVIEW

**Review by alamb — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -237,14 +237,14 @@ curl -v -G -d 'org=company' -d 'bucket=sensors' --data-urlencode 'sql_query=sele
 
 The HTTP API exposes a healthcheck endpoint at `/health`
 
-```shell
+```console
```
Comment (domodwyer) [inline README.md:240]: I didn't know this was a thing!

**Review by domodwyer — APPROVED** [review]
🐚

File: README.md (hunk 1 above)
Comment (mkmik) [reply README.md:240]: image

Comment (NGA-TRAN) [reply README.md:240]: Same to me. Have been googling but have not found anything useful. Do you have useful reading for us, @mkmik ?

Comment (mkmik) [reply README.md:240]: I don't know if there is any official documentation; GH uses this tool to render code (including markdown snippets): https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5314

Comment (NGA-TRAN) [reply README.md:240]: Yeah, I search lead me there, too :)

**Pull request merged**